mautic/mautic
# Support syncList form in Focus

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4768
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
If we are using syncList option from LeadField in forms. These options wasn't  loaded in Focus. 
This PR fix it. Issue reported here https://github.com/mautic/mautic/issues/4768

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Create custom lead field with select box and 3 options
2.  Create form with email and custom select box and enable option in propreties **Use assigned contact/company field's list choices.**
![image](https://user-images.githubusercontent.com/462477/38502451-b4d93c14-3c0f-11e8-999e-36ee535381b5.png)

3. Create focus with form
4. Preview focus and selectbox wasn't load options 

#### Steps to test this PR:
1. Repeat all steps
2. See selectbox work properly
